ci/lint.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: latest
```

Comment:
This should really be locked to major version, so use `1.18.4`

Reply:
linters could break going from 1 major go version to another

Reply:
Done